protos/feast/core/Feature.proto

```diff
@@ -44,5 +44,5 @@ message FeatureSpecV2 {
     string vector_search_metric = 6;
 
     // Field indicating the vector length
-    int32 vector_len = 7;
+    int32 vector_length = 7;
```

Comment:
technically this is a breaking change, so we should add a new protobuf here.

Reply:
Yup, the new protobuf is added , in this PR only